components/phpunit_bridge.rst

```diff
@@ -573,8 +573,12 @@ If you have installed the bridge through Composer, you can run it by calling e.g
 
 .. tip::
 
-    Set the ``SYMFONY_PHPUNIT_VERSION`` env var to e.g. ``5.5`` to change the
-    base version of PHPUnit to ``5.5`` instead of the default ``5.3``.
+    Set a real env var (not in a :doc:`dotenv </components/dotenv>` file)
+    ``SYMFONY_PHPUNIT_VERSION`` to e.g. ``5.5`` to change the base version
+    of PHPUnit to ``5.5`` instead of the default ``5.3``.
+
+    It's also possible to set this env var in the ``phpunit.xml.dist`` file
```

Comment:
I would recommend to use this first, because the version should be committed. Then telling a real env vars works too is nice.